syncd/VendorSai.cpp

```diff
@@ -657,11 +657,35 @@ sai_status_t VendorSai::bulkGet(
         _In_ sai_bulk_op_error_mode_t mode,
         _Out_ sai_status_t *object_statuses)
 {
+    MUTEX();
     SWSS_LOG_ENTER();
+    VENDOR_CHECK_API_INITIALIZED();
+
+    sai_bulk_object_get_attribute_fn ptr;
+
+    switch (object_type)
+    {
+        case SAI_OBJECT_TYPE_PORT:
+            ptr = m_apis.port_api->get_ports_attribute;
```

Comment:
since you are adding bulk support, add all apis here that support bulk ?

Reply:
Same as for SET API. I add here only what I tested and have unit tests for (to satisfy coverage requirement). If anyone wants to add more APIs they can be added seperatelly in which case a seperate unit testing code needs to be written.